docs/sources/helm-charts/mimir-distributed/_index.md

```diff
@@ -8,15 +8,18 @@ keywords:
   - Grafana Enterprise Metrics
   - Grafana metrics
 cascade:
-  # Used to generate relative URL paths to related documentation
   mimir_docs_version: "v2.9.x"
```

Comment:
I think we should rename this to MIMIR_DOCS_VERSION and have the same spelling everywhere for easy understanding and grep.